### PR TITLE
[master] logging fix

### PR DESCRIPTION
**Pull request by mjudeikis**
```release-note
- Fix logging
```

Fixes missing logs issue

### CODE REVIEW

File: pkg/startup/v7/startup.go [hunk 1]
```diff
@@ -119,7 +119,7 @@ func (s *startup) writeFiles(role api.AgentPoolProfileRole, w writers.Writer, ho
 			filepath = strings.TrimPrefix(filepath, "common")
 
 			filesToWrite[filepath] = string(b)
-
+			fileKeys = append(fileKeys, filepath)
```
Comment (asalkeld) [inline pkg/startup/v7/startup.go]: are you sure we should be changing this? we never create old versions (except in CI), they are there for hash comparisons really so you are now changing the old hashes..

File: pkg/cluster/hash_test.go [hunk 2]
```diff
@@ -36,34 +36,34 @@ func TestHashScaleSetStability(t *testing.T) {
 			{
 				role: api.AgentPoolProfileRoleMaster,
 				// this value should not change
```
Comment (asalkeld) [inline pkg/cluster/hash_test.go]: this ^
I don't think we should "fix" the old versions

Comment (mjudeikis) [reply pkg/cluster/hash_test.go]: I think this will be aligned with other PR's, no?

Comment (asalkeld) [reply pkg/cluster/hash_test.go]: i am ok with it..

**Comment by mjudeikis** [comment]
manually tagging

**Pull request merged**